### PR TITLE
Feature/allisonbhavsar/30 module api routes

### PR DESCRIPTION
**Pull request by allisonbhavsar**
### Administrative Info

Make sure your branch name conforms to: `<feature/staging/hotfix/...>/[username]/[Github Issue]-[3-4 word description separated by dashes]`.

### What issue(s) does this branch close?
#30 

### Changes

What changes did you make?

- modules and module_items db tables
- Added module and module items API routes

### Testing

How did you confirm your changes worked?

- Tested creating, updating, deleting, and getting class modules
- Tested creating, deleting, and getting module items

### Confirmation of Change

npm run test

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -16,6 +16,6 @@
     "jsx": "preserve",
     "incremental": true
   },
-  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx"],
+  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", "pages/api/class/[id]/modules"],
```
Comment (Anshul-Birla) [inline tsconfig.json:19]: is there a reason this file was added?

**Review by Anshul-Birla — CHANGES_REQUESTED** [review]
Just one super small question - other than that everything looks great!

**Review by nishantbalaji — APPROVED** [review]
lgtm!

**Review by Anshul-Birla — APPROVED** [review]
LGTM!

**Pull request merged**